prepare/cards/wnli.py

```diff
@@ -11,24 +12,18 @@
 card = TaskCard(
     loader=LoadHF(path="glue", name="wnli"),
     preprocess_steps=[
-        "splitters.small_no_test",
-        RenameFields(
-            field_to_field={
-                "sentence1": "text_a",
-                "sentence2": "text_b",
-            }
+        SplitRandomMix(
+            {"train": "train[95%]", "validation": "train[5%]", "test": "validation"}
```

Comment:
Isn't it the same as splitters.small_no_test? Why not using it?

Reply:
See this issue #925.    I think these catalog splitters are not clearly named  and hence not understood by users , especially in initial examples.